openscap.spec

```diff
@@ -82,6 +82,14 @@ The %{name}-scanner package contains oscap command-line tool. The oscap
 is configuration and vulnerability scanner, capable of performing
 compliance checking using SCAP content.
 
+%package        remediate-offline
```

Comment:
We don't ship the oscap-remediate-offline script, right? Shouldn't the package name be just `remediate` or `remediate-service`?

Reply:
Actually I don't like any combination with `remediate`. Because it might give an impression that the sub-package is needed for the scanner to be able to remediate the system. The feature of systemd that we utilize is called `system-update`. How about `openscap-system-update` sub-package?

Reply:
I'll put this question on the consilia.

Reply:
My point was to avoid confusion with the "oscap-remediate-offline" executable. But you have a good point that we can have a different confusion as well.